### PR TITLE
Added base level documentation

### PR DESCRIPTION
**Pull request by chums923**
Connected to #20 

Added doc subfiles

Finished first pass at documentation

Removed crashReport file

Fixed bug in MD

Fixed md bug

Fixed md bug

Fixed typos

Added basic documentation

### CODE REVIEW

File: locmap/docs/user.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+* Requires valid User ID when querying API
+
+***
+
+##`POST` /user/:userid/location
+Update user location
+
+###Parameters
+- **userId** _(required)_ - Id of current user
+- **request cache** _(required)_ - Current cache of user (from authorization step)
```
Comment (josalmi) [inline locmap/docs/user.md]: Why is this listed as a parameter? What does this actually mean? Now correct me if I'm wrong. When user authenticates, his/her data is stored in the backend's locmap cache variable. That doesn't actually mean that it is a parameter.

**Comment by Kauhsa** [comment]
I think the vertical rules are not needed – they look a bit distracting to me when looking at the rendered view.

File: locmap/docs/admin.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+***
+
+##`POST` /admin/:userId/accountRecovery
+Puts account into recovery mode
+
+###Parameters
+- **userId** _(required)_ - Id of current user (must be admin)
+- **request body** _(required)_ - Included in this body is:
+	#####body
```
Comment (Kauhsa) [inline locmap/docs/admin.md]: I'm not sure what the hashes are supposed to be, but it doesn't seem to render correctly at least in GitHub.

File: locmap/docs/admin.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+* Requires Admin Authentication to access any of the below methods
+
+***
+
+##`GET` /admin/:userId/crashReport/:osType/:year/:month
+Get crash reports for given OS and time period
+
+###Parameters
+- **userId** _(required)_ - Id of current user (must be admin)
+- **OS type** _(required)_ - Get crash reports from this OS
```
Comment (Kauhsa) [inline locmap/docs/admin.md]: Style is inconsistent with parameter name

**Comment by Kauhsa** [comment]
Looks very comprehensive, good job!

**Comment by chums923** [comment]
Okay I'll leave this up for now, but I'll make some changes when I get back to Boston, cancel this one, and make a new PR. As for the cache, I thought it was coming in through the request. Is that wrong/should we not be including it as a parameter. I just pretty much put anything in the parameter list that was in the code and started with "req."

**Pull request closed without merging**